playbooks/support/responding.md

```diff
@@ -11,6 +11,23 @@ respond to these issues.
   These automated system alerts _may_ be the first sign of a problem but don't _necessarily_ warrant treatment as
   immediate incidents. E.g., load-balancers may route around failing EC2 instances or auto-scaling may bring up
   capacity in response to load.
+- Monitor and acknowledge automated uptime incidents created by OpsGenie.
+
+  - For high-criticality systems, OpsGenie will open incidents if
```

Comment:
Does OpsGenie open an incident automatically or does it create an alert in response to failing uptime checks from Pingdom? I think it might be the latter.

Once acknowledged, if the responding on-call engineers decide to open an incident in response, there are ways within the OpsGenie UI to associate open alerts with the new incident. Might be worthwhile to explore and document as a follow-up.